<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>247</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>coder2countdown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Hours</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="hours_spinbox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Minutes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="minutes_spinbox">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Seconds</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="seconds_spinbox">
          <property name="maximum">
           <number>59</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pause_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Format string (&quot;{hh}&quot;, &quot;{mm}&quot;, &quot;{ss}&quot; will be replaced):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="format_string_edit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="finish_message_checkbox">
        <property name="text">
         <string>Show different text when countdown reaches zero</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Finish message:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="finish_message_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="filename_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="destination_button">
        <property name="text">
         <string>Choose Destination</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hours_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spinbox_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minutes_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spinbox_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>seconds_spinbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_spinbox_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finish_message_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_finish_message_checkbox_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>format_string_edit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_format_string_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_spinbox_valueChanged(int)</slot>
  <slot>on_finish_message_checkbox_toggled(bool)</slot>
  <slot>on_format_string_changed(QString)</slot>
 </slots>
</ui>
